pages/blog/release-notes-2.5.0.md

```diff
@@ -0,0 +1,145 @@
+
+We noticed that we overlooked and did not define the Operation Bindings Object in the case of IBM MQ. 
+In AsyncAPI 2.5.0, the definition is empty, but its name is reserved for future use to be consistent with the rest of the bindings.
+
+This change was contributed by [Maciej Urbańczyk](https://github.com/magicmatatjahu). For more detail, see [Maciej's `/spec #840` pull request](https://github.com/asyncapi/spec/pull/840).
+
+## Added new Google Cloud Pub/Sub Bindings
+
+The specification is now extended to support another custom protocol through the bindings feature:
+
+Google Cloud Pub/Sub, thanks to [Jeremy Whitlock](https://github.com/whitlockjc). For more details, check out this [pull request](https://github.com/asyncapi/spec/pull/836) and the [binding definition](https://github.com/asyncapi/bindings/tree/master/googlepubsub).
+
+## Clarifications
+
+Here are notable clarifications that got into the AsyncAPI 2.5.0 release. Here is the list of notable clarifications that may have an impact on the tooling authors:
+
+**ChannelItem.$ref resolution rules**
+
+All references of `MUST be in the format of a URL` sentence have been substituted with `This MUST be in the form of an absolute URL`.
+The meaning of `absolute URL` is now defined by [RFC3986, section 4.3](https://datatracker.ietf.org/doc/html/rfc3986#section-4.3).
+
+**Clarification around URL resolution rules**
+
+The resolution rules around `ChannelItem.$ref` field were not clearly defined. AsyncAPI 2.5.0 comes with explicit clarification that the resolution is done as defined by the [JSON Reference](https://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03).
+
+## Tooling support
+
+The following official AsyncAPI tools are already updated to support the 2.5.0 specification version:
+- JSON Schema that supports validation of AsyncAPI documents is updated in our [specification JSON schema repository](https://github.com/asyncapi/spec-json-schemas). We updated the **@asyncapi/specs** package on NPM to version `3.2`, containing the JSON Schema document for AsyncAPI Spec 2.5.0.
+- [JavaScript Parser](https://github.com/asyncapi/parser-js/) can be used to parse and validate 2.5.0 documents. Upgraded to `the 1.17.0` version.
+- [AsyncAPI React Component](https://github.com/asyncapi/asyncapi-react) allows you to render and validate the documentation of your asynchronous API provided in the AsyncAPI specification format. Upgraded to the `v1.0.0-next.43` version.
+- [HTML template](https://github.com/asyncapi/html-template) uses the latest **@asyncapi/react-component** package. Upgraded to the `0.26.0` version.
+- [JavaScript Converter](https://github.com/asyncapi/converter-js/) enables conversion from any AsyncAPI version into the 2.5.0 version of the spec. Upgraded to the `1.1.0` version.
+- [Generator](https://github.com/asyncapi/generator/) uses the latest @asyncapi/parser package, so while generating output, it can validate 2.5.0 documents. Upgraded to the `1.10.0` version.
```

Comment:
```suggestion
- [Generator](https://github.com/asyncapi/generator/) uses the latest @asyncapi/parser package, so while generating output, it can validate 2.5.0 documents. Upgraded to the `1.10.0` version.
- [Modelina](https://github.com/asyncapi/modelina/) can be used to generate typed models. Upgraded to the `1.0.0-next.11` version.
```

Reply:
How come? 🤔

Reply:
Ahh, only code owner for markdown files 😆

Reply:
Also `generator` is released as `1.9.12` not `1.10.0` cc @derberg

Reply:
oh crap, sorry

@char0n we need update, so please take suggestions from @jonaslagoni and update info about generator as @magicmatatjahu wrote

Reply:
There are so many tools missing from the list that are still in the process of being updated. I don't see any benefit in only highlighting a small section of the tools. It feels rushed which I don't think is good 😅 But of course the alternative is that we give tools a couple of days before releasing the release notes. Which also is sub-optimal.

We could list all the tools that are in-progress of being updated and their PRs? 🤔

Reply:
Studio and CLI is also missing.

Reply:
Studio is highlighted under list :)

Reply:
> There are so many tools missing from the list that are still in the process of being updated

which ones?

Reply:
For CLI https://github.com/asyncapi/cli/releases/tag/v0.25.0

Reply:
> Studio is highlighted under list :)

Ahh, just hiding in the end 😆 I even checked for it before writing...

Reply:
We also have:
- Bundler https://github.com/asyncapi/bundler/releases/tag/v0.2.3
- Cupid https://github.com/asyncapi/cupid/releases/tag/v0.6.20
- Optimizer https://github.com/asyncapi/optimizer/releases/tag/v0.1.11
- Glee https://github.com/asyncapi/glee/releases/tag/v0.10.23

and so on and so on.

Diff had it's dependency updated as a `chore` not sure what that's about https://github.com/asyncapi/diff/pull/117 should have been a `fix` I think.

Reply:
Server API https://github.com/asyncapi/server-api/releases/tag/v0.11.0

Reply:
> Diff had it's dependency updated as a chore not sure what that's about https://github.com/asyncapi/diff/pull/117 should have been a fix I think.

diff do not use parser, it is there only in devdependencies for testing

Reply:
> diff do not use parser, it is there only in devdependencies for testing

Ahh right, makes sense 👍